lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,388 @@
+                    <a class="cover">
+                        <img src="<%- model.course_image_url.small %>" class="course-image"
+                             alt="<%- courseCoverImageText %>"/>
+                    </a>
+                    <% } %>
+                    <% if (settings.enable_verified_certificates) { %>
+                    <span class="sts-enrollment" title="<%- model.course_verified_certs.enrollment_title %>">
+                          <span class="label"><%- gettext("Enrolled as: ") %></span>
+                        <% if (model.course_verified_certs.show_image){ %>
+                        <img class="deco-graphic" src="/static/images/verified-ribbon.png"
+                             alt="<%- model.course_verified_certs.image_alt %>"/>
+                        <% } %>
+                        <div class="sts-enrollment-value"><%- model.course_verified_certs.enrollment_value %></div>
+                        </span>
+                    <% } %>
+                </div>
+                <div class="wrapper-course-details">
+                    <h3 class="course-title">
+                        <% if (model.show_courseware_link) {
+                             if (!model.is_course_blocked) { %>
+                                <a data-course-key="<%- model.course_key %>"
+                                   href="<%- model.course_info_url %>"><%- model.course_name %></a>
+                            <% } else { %>
+                                <a class="disable-look" data-course-key="<%- model.course_key %>"><%- model.course_name %></a>
+                            <% }
+                            } else { %>
+                        <span><%- model.course_name %></span>
+                        <% } %>
+                    </h3>
+
+                    <div class="course-info">
+                        <span class="info-university"><%- model.course_university_about_section %> - </span>
+                        <span class="info-course-id"><%- model.course_number %></span>
+                        <span class="info-date-block" data-tooltip="Hi">
+                        <% if (model.has_course_ended) { %>
+                            <%= _.sprintf(gettext("Ended - %(endDate)s"), { endDate:model.short_end_date}) %>
+                        <% } else if (model.has_course_started) { %>
+                            <%= _.sprintf(gettext("Started - %(startDate)s"), { startDate:model.short_start_date}) %>
+                        <% } else if (model.start_date_is_still_default) { %>
+                            <%- gettext("Coming Soon") %>
+                        <% } else if (model.starts_within_5_days) { %>
+                            <%= _.sprintf(gettext("Starts - %(dayAndTime)s"), { dayAndTime:model.day_and_time}) %>
+                        <% } else { %>
+                            <%= _.sprintf(gettext("Starts - %(startDate)s"), { startDate:model.short_start_date}) %>
+                        <% } %>
```

Comment:
Can you please move this logic into the View so a string is passed to the template and just inserted here.